apps/src/dance/danceRedux.ts

```diff
@@ -166,14 +168,22 @@ const danceSlice = createSlice({
     setCurrentSongMetadata: (state, action: PayloadAction<SongMetadata>) => {
       state.currentSongMetadata = action.payload;
     },
-    setCurrentAiModalField: (
+    setAiOutput: (state, action: PayloadAction<AiOutput>) => {
+      state.aiOutput = action.payload;
+    },
+    openAiModal: (
       state,
-      action: PayloadAction<GoogleBlockly.Field | undefined>
+      action: PayloadAction<{
+        modalField: GoogleBlockly.Field;
```

Comment:
super nit but while we're here, can we also switch this from `GoogleBlockly.Field` to just `Field`? I think we should be able to `import {Field} from 'blockly'`